README.md

```diff
@@ -25,6 +25,12 @@ npm test
 ```
 This will run ganache-cli, compile, migrate and run all tests.
 
+### Docker
+Arc has a prebuilt docker image that makes development environments consistent and cross-platform.
+1. Run `docker run -it -v <path to repo>:/home/arc devmatantsu/arc`.
```

Comment:
Please add in the line above link for docker installation, so it will be clear one must install docker before running this.
In addition, you might want to say that it might require admin permission (sudo on unix).
Thirdly, the package cannot be under your account, it looks bad. We need a daostack account.